src/model/table.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+from model.index import *
+
+
+class TableMetaData:
```

Comment:
why the file name is table and class name TableMetadata, it shoud be `table_metadata.py`